packages/opentelemetry-core/src/platform/browser/environment.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DEFAULT_ENVIRONMENT,
+  ENVIRONMENT,
+  ENVIRONMENT_MAP,
+  parseEnvironment,
+} from '../../utils/environment';
+
+/**
+ * Gets the environment variables
+ */
+export function getEnv(): ENVIRONMENT {
+  const globalEnv = parseEnvironment((window as unknown) as ENVIRONMENT_MAP);
```

Comment:
instead of asserting `window` twice can we do `const _window = window as typeof window & ENVIRONMENT_MAP` like we did in the API package for global?

Reply:
ENVIRONMENT_MAP should be made partial. The types will not force you to check if a value exists in this current setup.

Reply:
not sure if I understand correctly your last sentence about partial ?